models/reviews.model.js

```
@@ -1,5 +1,24 @@
 const db = require("../db/connection.js")
 
+exports.insertCommentWithID = (req) => {
+    if((req.params.review_id).match(/^\d+$/)){
```

PSQL will handle this for us, so we don't need to check whether a review_id is a number. If it isn't, PSQL should throw a particular error message that we can make use of.